modules/kmm-applying-tolerations-to-kernel-module-pods.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-applying-tolerations-to-kernel-module-pods_{context}"]
+= Applying tolerations to kernel module pods 
+
+Taints and tolerations consist of `effect`, `key`, and `value` parameters. Tolerations include additional `operator` and `tolerationSeconds` parameters. 
+
+`effect`:: Indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are  `NoSchedule`, `PreferNoSchedule`, or `NoExecute`.
+
+`key`:: The taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, `operator` must be `Exists`. This combination means to match all values and all keys.
+
+`value`:: The taint value the toleration matches to. If `operator` is `Exists``, the value should be empty, otherwise just a regular string.
+
+`operator`:: Represents a key's relationship to the value. Valid operators are `Exists` and the default value `Equal`. `Exists` is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+
+`tolerationSeconds`:: Represents the period of time the toleration (which must be of effect NoExecute, otherwise this field is ignored) tolerates the taint. By default, it is not set, which means tolerate the taint forever (do not evict). Zero and negative values will be treated as 0 (evict immediately) by the system.
+
+.Example taint in a module specification
+[source,yaml]
+----
+apiVersion: kmm.sigs.x-k8s.io/v1beta1
+kind: Module
+metadata:
+  name: <my_kmod>
+spec:
+  ...
+  taints:
+    effect: NoSchedule
+    key: key1
+    value: value1
```

Comment:
Taints are applied to a `node` and not to a `Module`.

Reply:
@ybettan Not sure what you mean by this. Do you mean I should change lines 21 and 25 from `Module` to `Node`?

Reply:
I mean that you created an example that won't work as there is no such field `taints` in the `Module` object. `taints` are only existing on `Node` objects. You can find a taint example (applied to a node) [here](https://docs.openshift.com/container-platform/4.18/nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations).

Reply:
LGTM